syntax = "proto3";

package orderservice;
message OrderRequest {
    int32 user_id = 1;
    int32 product_id = 2;
}
message OrderResponse {
    string message = 1;
}
service OrderService {
    rpc CreateOrder (OrderRequest) returns (OrderResponse);
}